NuGet.config

```diff
@@ -40,6 +40,18 @@
     <!--  End: Package sources from dotnet-msbuild -->
     <!--  Begin: Package sources from dotnet-roslyn-analyzers -->
     <!--  End: Package sources from dotnet-roslyn-analyzers -->
+    <!--  Begin: Package sources from dotnet-runtime -->
+    <!--  End: Package sources from dotnet-runtime -->
+    <!--  Begin: Package sources from dotnet-templating -->
+    <!--  End: Package sources from dotnet-templating -->
+    <!--  Begin: Package sources from dotnet-windowsdesktop -->
+    <!--  End: Package sources from dotnet-windowsdesktop -->
+    <!--  Begin: Package sources from dotnet-aspnetcore -->
+    <!--  End: Package sources from dotnet-aspnetcore -->
+    <!--  Begin: Package sources from dotnet-msbuild -->
+    <!--  End: Package sources from dotnet-msbuild -->
+    <!--  Begin: Package sources from dotnet-roslyn-analyzers -->
+    <!--  End: Package sources from dotnet-roslyn-analyzers -->
```

Comment:
@mmitche I'm assuming we shouldn't be getting these duplicated comments here?

Reply:
Nope. @MattGal Any idea why the comments are getting duplicated?

Reply:
I definitely caused this, will address via https://github.com/dotnet/core-eng/issues/11362